1_fetch/src/download_sb_data.r

```diff
@@ -5,13 +5,12 @@
   library(sbtools)
   library(whisker)
   
-  fetch_data <- function() {
-    project_output_dir <- '1_fetch/out'
+  fetch_data <- function(project_output_dir) {
     #create directory
-    dir.create(project_output_dir)
+    dir.create(project_output_dir, showWarnings = FALSE)
```

Comment:
Since you're now passing the file name, rather than the directory name, you'll want to extract the directory name (`dirname(filepath)`) an pass that to `dir.create()` instead of the full filepath.